leaflet/index.d.ts

```diff
@@ -1524,6 +1524,26 @@ declare namespace L {
         export const svg: boolean;
     }
 
+    export namespace Util {
+        export function extend(dest: {}, src?: {}): {};
```

Comment:
I suggest changing `{}` to `any` since `{}` is pretty unusual in the definitions I've looked at so far.